src/odb/src/db/dbPowerSwitch.cpp

```diff
@@ -132,6 +142,13 @@ dbOStream& operator<<(dbOStream& stream, const _dbPowerSwitch& obj)
   stream << obj._on_state;
   stream << obj._control_net;
   stream << obj._power_domain;
+  // User Code Begin <<
+  if (obj.getDatabase()->isSchema(db_schema_upf_power_switch_mapping)) {
```

Comment:
There's no need to check on output as this will always be true.  We don't support writing in old formats, only reading them.

Reply:
got it, fixed

Reply:
I believe you did this backwards.  The operator>> needs the check not the operator<<

Reply:
updated